### ISSUE TITLE
Need a Service selection Annotation config as well

### ISSUE THREAD

**Issue by linsite** [kube-vip/kube-vip#409]
**Is your feature request related to a problem? Please describe.**
Kube-vip is designed to balance load for all LoadBalance Service that without the ignorance Annotation by default. That may be confused that a Service doesn't need to be balanced but has to update its definition to add the Annotation.

**Describe the solution you'd like**
Is it reasonable to implement the opposite way to only proxy the Services with a specific Annotation?

**Describe alternatives you've considered**
The config can be more flexible to support both situations.

**Additional context**
Nope.

**Comment by thebsdbox** [comment]
This is probably negated through the use of the `loadBalancerClass` added recently.

**Comment by linsite** [comment]
Thanks a lot, I'll take a look. :)

**Comment by linsite** [comment]
Yes, that's what I want. I think we can add some docs updating for new users. FYI, I'd be happy to do that if it's neccesarry.

**Comment by thebsdbox** [comment]
That would be great. All of the docs are in the website repo from now on.
